components/prism-pascal.js

```diff
@@ -5,9 +5,17 @@
 */
 
 Prism.languages.pascal = {
+	'asm': {
+		pattern: /(asm)([\s\S]+?)(end)/i,
+		lookbehind: true,
+		inside:{keyword:/end/i,punctuation:/;$/i}
+	},	
+	'directive': [
+		/\{\$[\s\S]+?\}/
+	],	
```

Comment:
```suggestion
	'directive': /\{\$[\s\S]+?\}/,
```

Reply:
In fact, that's how it should be.
It was the first time I analyzed the prism.js code
I need it for my website and it wasn't working properly so I'm trying to improve.